aspnet/mobile/native-mobile-backend.rst

```diff
@@ -1,11 +1,170 @@
 
-|stub-icon| Creating Backend Services for Native Mobile Applications
-====================================================================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+Mobile apps can easily communicate with ASP.NET Core backend services.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/mobile/native-mobile-backend/sample>`__
+
+The Sample App
+--------------
+
+This tutorial demonstrates how to create backend services using ASP.NET Core MVC to support native mobile apps. It uses the `Xamarin Forms TodoRest app <https://developer.xamarin.com/guides/xamarin-forms/web-services/consuming/rest/>`_ as its native client, which includes separate native clients for Android, iOS, Windows Universal, and Window Phone devices. The sample application includes an ASP.NET Web API 2 services project, which this article's ASP.NET Core MVC app replaces (with no changes required by the client).
+
+.. image:: native-mobile-backend/_static/todo-android.png
+
+.. note:: Securing web APIs is not covered in this article, but will be covered in follow-up documentation.
+
+Features
+^^^^^^^^
+
+The TodoRest app supports listing, adding, deleting, and updating To-Do items. Each item has an ID, a Name, Notes, and a property indicating whether it's been Done yet.
+
+The main view of the items, as shown above, lists each item's name and indicates if it is done with a checkmark.
+
+Tapping the `+` icon opens an add item dialog:
+
+.. image:: native-mobile-backend/_static/todo-android-new-item.png
+
+Tapping an item on the main list screen opens up an edit dialog where the item's Name, Notes, and Done settings can be modified, or the item can be deleted:
+
+.. image:: native-mobile-backend/_static/todo-android-edit-item.png
+
+This sample works with a read-only set of backend services by default. To test it out yourself against the ASP.NET Core app created in the next section running on your computer, you'll need to update the app's ``RestUrl`` constant. Navigate to the ``TodoREST`` project and open the `Constants.cs` file. Replace the ``RestUrl`` with a URL that includes your machine's IP address (not localhost or 127.0.0.1, since this address is used from the device emulator, not from your machine). Include the port number as well (5000). In order to test that your services work with a device, ensure you don't have an active firewall blocking access to this port.
+
+.. code-block:: csharp
+
+    // URL of REST service (Xamarin ReadOnly Service)
+    //public static string RestUrl = "http://developer.xamarin.com:8081/api/todoitems{0}";
+    
+    // use your machine's IP address
+    public static string RestUrl = "http://192.168.1.207:5000/api/todoitems/{0}";
+
+
+Creating the ASP.NET Core MVC Project
+-------------------------------------
+
+Create a new ASP.NET Core MVC project in Visual Studio. Choose the Web API template and No Authentication.
```

Comment:
screen shot shows Host in the cloud cleared, but it's selected by default the first time the dialog appears, so it might be worth mentioning that it should be cleared.